vms/proposervm/pre_fork_block.go

```diff
@@ -248,6 +248,7 @@ func (b *preForkBlock) buildChild(ctx context.Context) (Block, error) {
 		zap.Stringer("blkID", blk.ID()),
 		zap.Stringer("innerBlkID", innerBlock.ID()),
 		zap.Uint64("height", blk.Height()),
+		zap.Uint64("pChainHeight", pChainHeight),
```

Comment:
This was useful for debugging... I think it makes sense for it to be logged normally.